source/libmvvm_model/mvvm/utils/binutils.cpp

```diff
@@ -0,0 +1,112 @@
+#include "binutils.h"
+
+
+std::string file_check(std::string filename)
```

Comment:
The function is returning string which is not clear how to use. Introduce function IsASCIIFile() which will return `true` only if your analysis succeeded, and ASCII nature was defined. Otherwise `false`.